CHANGELOG/CHANGELOG-3.4.md

```diff
@@ -10,14 +10,16 @@ Previous change logs can be found at [CHANGELOG-3.3](https://github.com/etcd-io/
 - Fix [server/embed: fix data race when starting both secure & insecure gRPC servers on the same address](https://github.com/etcd-io/etcd/pull/15518)
 - Add [`etcd --tls-min-version --tls-max-version`](https://github.com/etcd-io/etcd/pull/15486) to enable support for TLS 1.3.
 - Fix [server/auth: disallow creating empty permission ranges](https://github.com/etcd-io/etcd/pull/15621)
-
-### Go
-- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15333).
-- Compile with [Go 1.19+](https://go.dev/doc/devel/release#go1.19)
+- Upgrade [golang.org/x/net to v0.7.0](https://github.com/etcd-io/etcd/pull/15333).
+- Add [`etcd --listen-client-http-urls`](https://github.com/etcd-io/etcd/pull/15620) flag to support separating http server from grpc one, thus giving full immunity to [watch stream starvation under high read load](https://github.com/etcd-io/etcd/issues/15402).
 
 ### Package `clientv3`
 - Reverted the fix to [auth invalid token and old revision errors in watch](https://github.com/etcd-io/etcd/pull/15542).
 
+### Go
+- Require [Go 1.19+](https://github.com/etcd-io/etcd/pull/15337).
+- Upgrade [Go to 1.19.6](https://github.com/etcd-io/etcd/pull/15333).
```

Comment:
```suggestion
- Upgrade [Go to 1.19.7](https://github.com/etcd-io/etcd/pull/15429).
```

Or change it to 1.19.8 as I bumped golang to 1.19.8 in https://github.com/etcd-io/etcd/pull/15652. 

Or Let me update it in a separate PR if you want to keep it as it's for now.

Reply:
done